### DISCUSSION TITLE
Question about the 200amp connectors and gain_pga

### DISCUSSION THREAD

**Discussion by ideal2545** [CircuitSetup/Expandable-6-Channel-ESP32-Energy-Meter#55] [Q&A]
Hi sorry for openening two discussions!

I am using both the 100a/50ma clamps and the 200A/100ma clamps. In order to avoid having to manually calibrate I made sure to pick up everything from the site (except for the power supply which I sourced the exact same part no).  I was looking at the latest 6 channel example config and I noticed that the current transformer recommendations for both the 100amp and the 200amp are "27518" I just wanted to confirm that wasn't a typo. 

The second question I have is from that config I looked at gain_pga and see its 1X. I was reading the esphome docs and it suggets that you should use 2x if using the 100amp clamps, 4x for 100a-200amps. I was curious as to why the example doc uses 1X? is it more for uniformity and then adjustment at the gain_ct? 

Thank you I appreciate the information, I have some experience with esphome configs but this is a new arena for me so just trying to learn. 

Thanks!
Jon

**Comment by tolrahC** [comment]
Hi @ideal2545 
Did you ever figure this out?  I'm not sure if I understand this correctly and it doesn't have much documentation on this.

**Reply by ideal2545** [reply]
I did look in the discussions area I posted my code there

**Reply by nareddyt** [reply]
Hi @ideal2545 , can you link directly to that code? I don't see it in discussions anywhere
